cogs/make_member.py

```diff
@@ -153,17 +164,13 @@ async def make_member(self, ctx: TeXBotApplicationContext, group_member_id: str)
 
         guild_member_ids: set[str] = set()
 
-        request_headers: dict[str, str] = {
-            "Cache-Control": "no-cache",
-            "Pragma": "no-cache",
-            "Expires": "0",
-        }
-        request_cookies: dict[str, str] = {
-            ".ASPXAUTH": settings["MEMBERS_LIST_URL_SESSION_COOKIE"],
-        }
-        async with aiohttp.ClientSession(headers=request_headers, cookies=request_cookies) as http_session:  # noqa: E501, SIM117
-            async with http_session.get(url=settings["MEMBERS_LIST_URL"]) as http_response:
-                response_html: str = await http_response.text()
+        http_session: aiohttp.ClientSession = aiohttp.ClientSession(
+            headers=REQUEST_HEADERS,
+            cookies=REQUEST_COOKIES,
+        )
+
```

Comment:
Newlines should not be placed between variable declaration & use within a `with` block